study-builder/studies/brain/about-child.conf

```diff
@@ -163,6 +154,109 @@
+                  "templateText": "$prompt_PARENT_CHILD",
+                  "variables": [
+                    {
+                      "name": "prompt_PARENT_CHILD",
+                      "translations": [
+                        {
+                          "language": "en",
+                          "text": "My child has been diagnosed with a primary brain cancer, and we are  filling out this survey together."
+                        }
+                      ]
+                    }
+                  ]
+                },
+                "detailLabelTemplate": null,
+                "allowDetails": false,
+                "exclusive": false
+              },
+              {
+                "stableId": "PARENT",
+                "optionLabelTemplate": {
+                  "templateType": "TEXT",
+                  "templateCode": null,
+                  "templateText": "$prompt_PARENT",
+                  "variables": [
+                    {
+                      "name": "prompt_PARENT",
+                      "translations": [
+                        {
+                          "language": "en",
+                          "text": "My child has been diagnosed with a primary brain cancer, and I am filling out this survey on their behalf."
+                        }
+                      ]
+                    }
+                  ]
+                },
+                "detailLabelTemplate": null,
+                "allowDetails": false,
+                "exclusive": false
+              },
+            ],
+            "questionType": "PICKLIST",
+            "stableId": "WHO_FILLING",
+            "isRestricted": false,
+            "isDeprecated": false,
+            "promptTemplate": {
+              "templateType": "HTML",
+              "templateCode": null,
+              "templateText": "$prompt_WHO_FILLING",
+              "variables": [
+                {
+                  "name": "prompt_WHO_FILLING",
+                  "translations": [
+                    {
+                      "language": "en",
+                      "text": "Who is filling out this survey?"
+                    }
+                  ]
+                }
+              ]
+            },
+            "additionalInfoHeaderTemplate": null,
+            "additionalInfoFooterTemplate": null,
+            "validations": [],
+            "hideNumber": false
+            "validations": [
+              {
+                "ruleType": "REQUIRED",
+                "allowSave": true,
+                "hintTemplate": {
+                  "templateType": "TEXT",
+                  "templateCode": null,
+                  "templateText": "$WHO_FILLING_req_hint",
+                  "variables": [
+                    {
+                      "name": "WHO_FILLING_req_hint",
+                      "translations": [
+                        {
+                          "language": "en",
+                          "text": "Who is filling out this survey is required"
```

Comment:
This sounds a little awkward. Is this what study staff wants to say? Maybe just `Please select an option` ?